wazo_call_logd/cel_interpretor.py

```diff
@@ -353,7 +353,8 @@ def interpret_mixmonitor_start(self, cel, call):
         if matches := RECORDING_PATH_REGEX.match(extra['filename']):
             recording_uuid_str = matches.group(2)
             recording.uuid = uuid.UUID(recording_uuid_str)
-
+        else:
```

Comment:
If the filename does not match the regex we could end up with 2 recordings that have the same filename and different uuid. I don't mind that but we need to make sure it does not break the call log generation.